site/src/components/SSHButton/SSHButton.tsx

```diff
@@ -83,9 +89,47 @@ export const SSHButton: React.FC<React.PropsWithChildren<SSHButtonProps>> = ({
         </Stack>
 
         <HelpTooltipLinksGroup>
-          <HelpTooltipLink href="https://coder.com/docs/coder-oss/latest/install">
-            Install Coder CLI
-          </HelpTooltipLink>
+          <Link
+            onClick={() => setDownloadCLIOpen(!downloadCLIOpen)}
+            className={styles.link}
+          >
+            {downloadCLIOpen ? (
+              <ExpandMoreIcon className={styles.linkIcon} />
+            ) : (
+              <ChevronRightIcon className={styles.linkIcon} />
```

Comment:
This looks really good, but I think maybe it should open a popup or something instead with some more details as to how to install the CLI. We should recommend GitHub releases by default, and only recommend this for airgapped users.

We also have other ways of installing Coder like the install script, etc.

Reply:
@BrunoQuaresma ^